packages/trader/src/AppV2/Components/TradeParameters/Duration/duration.tsx

```diff
@@ -125,10 +126,14 @@ const Duration = observer(({ is_minimized }: TDurationProps) => {
             const error_obj = proposal_info[contract_type_object[0]] || validation_errors?.duration?.[0];
             if (error_obj?.error_field === 'duration') {
                 addSnackbar({
-                    message: <Localize i18n_default_text={error_obj.message} />,
```

Comment:
`<Localize />` is not working with variables, so I removed wrapper. The value should be already translated as it's comming from BE